internal/rpcserver/rpcserver.go

```diff
@@ -3849,21 +3875,50 @@ func handleGetWorkRequest(ctx context.Context, s *Server) (interface{}, error) {
 	return reply, nil
 }
 
+// minInt is a helper function to return the minimum of two ints.  This avoids
+// the need to cast to floats.
```

Comment:
not sure what floats have to do with this?

Reply:
Both of the standard lib [math.Min](https://pkg.go.dev/math#Min) and [math.max](https://pkg.go.dev/math#Max) take floats.